### PR TITLE
simplify libtest queue utils

### PR DESCRIPTION
**Pull request by Kokan**
While I checked the disk queue related tests, I had some time to kill so I thought this changes improve the code.
Now the queue util based functions do not depend on `syslogformat` module.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by MrAnno** [comment]
Could you rephrase the last commit message to clarify that it's about a test case, please?

logqueue: test with empty messages,
use empty messages for queue tests,
simplify test case,
etc.

**Comment by Kokan** [comment]
Sure.

**Comment by kira-syslogng** [comment]
Build SUCCESS
